prometheus.yml

```diff
@@ -0,0 +1,8 @@
+scrape_configs:
+  - job_name: 'deku'
+    scrape_interval: 1s
+    static_configs:
+      # TODO: currently these are hard-coded for development/local benchmarking
+      # But we might want to do this more dynamically. Maybe Prometheus has some
+      # dynamic config?
```

Comment:
It depends on the type or orchestration you're using, for this case you may want to try it for Docker.

https://prometheus.io/docs/prometheus/latest/configuration/configuration/#docker_sd_config